benchmark/http/headers.js

```diff
@@ -14,7 +14,7 @@ function main({ len, n }) {
     'Transfer-Encoding': 'chunked',
   };
 
-  const Is = [ ...Array(n / len).keys() ];
+  const Is = [ ...Array(Math.max(n / len, 1)).keys() ];
```

Comment:
Would you be so kind and add a `TODO` that links to https://github.com/nodejs/node/issues/26425?

```suggestion
  // TODO(BridgeAR): Change this benchmark to use grouped arguments when
  // implemented. https://github.com/nodejs/node/issues/26425
  const Is = [ ...Array(Math.max(n / len, 1)).keys() ];
```